src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
If you have a default constructor, your variables should have default values, otherwise, it could cause NullPointerException. Or you can remove the empty constructor and use only with parameters.